crates/bevy_animation/src/lib.rs

```diff
@@ -525,12 +525,13 @@ impl Hash for AnimationTargetId {
 /// Note that each entity can only be animated by one animation player at a
 /// time. However, you can change [`AnimationTarget`]'s `player` property at
 /// runtime to change which player is responsible for animating the entity.
-#[derive(Clone, Copy, Component, Reflect)]
+#[derive(Clone, Copy, Component, Reflect, IterEntities)]
 #[reflect(Component, MapEntities)]
 pub struct AnimationTarget {
     /// The ID of this animation target.
     ///
     /// Typically, this is derived from the path.
+    #[iter_entities(ignore)]
```

Comment:
Do we need to add this to each field without entities?

Reply:
This is the approach taken by the `Bundle`, `Reflect`, and probably other derives, right? Default-in, opt-out?

Yeah, the macro relies on the fields implementing `IterEntities` themselves, which can't be known just from the AST.

One alternative could be to try to infer which fields are entity-containing by the presence of `Entity` in their type, but then I think we'd need an attribute to opt-in for fields that aren't inferrable and also an opt-out for fields that are false-positives.

Another could be to invert the attribute and mark _only_ the entity-containing fields. Hard to say if that would require more or fewer attributes across non-core crates, but it would definitely require more in the ones in this repo.

Reply:
Got it. I think it make sense. At least I don't see a better option :)